examples/Fruit/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-fruit)
+
+hunter_add_package(Fruit)
```

Comment:
Please use `# DOCUMENTATION_START {` `# DOCUMENTATION_END }` to mark the code that will be part of documentation:
* https://github.com/ruslo/hunter/blob/21b692d45d87a0bbfa55604604852b7f6655a993/examples/foo/CMakeLists.txt#L12-L18